repos/.leapp/info

```diff
@@ -0,0 +1 @@
+{"name": "repos", "id": "411897a6-6503-46cd-be5b-b29c5154e0c0"}
```

Comment:
the name repos does not say so much. Imagine that someone else use the same name of the repo. so there will be a collision. regarding the git reponame, i would expect something like
* `system_upgrade_supplements`

Also, do not have a repo in directory "repos". The directory suggests I should be able to find more repositories inside possibly in fugure. put the directory into a separate dir, e.g. "repos/upgrade/supplementary".

Also the repository should have set dependency on the system_upgrade_common repository, which is not set currently based on this.

Reply:
Now I understand the structure of leapp-repositry, thanks for the correcting.

I created `system_upgrade/supplements` with repo link to `system_upgrade_common`